apps/explorer/src/components/ownedobjects/views/ObjectFieldsCard.tsx

```diff
@@ -0,0 +1,249 @@
+                                                {filteredFieldNames.length}
+                                                {filteredFieldNames.length === 1
+                                                    ? ' Result'
+                                                    : ' Results'}
+                                            </div>
+                                        ) : (
+                                            <div className="px-3.5 pt-2 text-center text-body italic text-gray-70">
+                                                No results
+                                            </div>
+                                        )}
+                                        {filteredFieldNames?.map(([name]) => (
+                                            <Combobox.Option
+                                                key={name}
+                                                value={name}
+                                                className="list-none md:min-w-fit"
+                                            >
+                                                {({ active }) => (
+                                                    <button
+                                                        type="button"
+                                                        className={clsx(
+                                                            'mt-0.5 block w-full cursor-pointer rounded-md border px-1.5 py-2 text-left text-body',
+                                                            active
+                                                                ? 'border-transparent bg-gray-40 text-steel-darker'
+                                                                : 'border-transparent bg-white font-medium text-steel-darker'
+                                                        )}
+                                                    >
+                                                        {name}
+                                                    </button>
+                                                )}
+                                            </Combobox.Option>
+                                        ))}
+                                    </Combobox.Options>
+                                </Combobox>
+                                <div className="max-h-[600px] min-h-full overflow-auto overflow-x-scroll py-3">
+                                    <VerticalList>
+                                        {fieldsNames.map(([name, value]) => (
+                                            <div
+                                                key={name}
+                                                className="mx-0.5 mt-0.5 md:min-w-fit"
+                                            >
+                                                <ListItem
+                                                    active={
+                                                        activeFieldName === name
```

Comment:
Should probably also expand the disclosure right?

Reply:
Yes, I will open another PR with the feature